src/DotVVM.Framework/tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "outFile": "Resources/Scripts/DotVVM.js",
     "declaration": true,
     "strictNullChecks": true,
-    "lib": ["dom", "es2015.promise", "es5"]
+    "lib": ["dom", "es2015.promise", "es5", "es2015.symbol"]
```

Comment:
What's the purpose of including `es2015.symbol`? Symbols are not supported by IE.